src/App.vue

```diff
@@ -2,3 +2,23 @@
 <template>
   <RouterView />
 </template>
+
+<script setup>
+import useLocalStorage from 'src/compositionFunctions/useLocalStorage'
+import { useQuasar } from 'quasar'
+import { ref, onMounted } from 'vue'
+import { useRouter } from 'vue-router'
+const router = useRouter()
+const { retrieveUserData } = useLocalStorage()
```

Comment:
can be simplified as const { retrieveUserData, retrieveDarkMode } = useLocalStorage